File: problemas/ad-hoc/1794/1794.jl
using Printf

n = parse(Int, readline())

numeros = [parse(Int, x) for x in split(readline())]
la = numeros[1]
lb = numeros[2]

numeros = [parse(Int, x) for x in split(readline())]
sa = numeros[1]
sb = numeros[2]

if (la <= n && n <= lb && sa <= n && n <= sb)
  @printf "possivel\n"
else
  @printf "impossivel\n"
end

File: problemas/ad-hoc/1794/1794.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine());
        List<int> L = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();
        List<int> S = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();

        if(L[0] <= N && N <= L[1] && S[0] <= N && N <= S[1]){
            Console.WriteLine("possivel");
        }else{
            Console.WriteLine("impossivel");
        }
    }
}